src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1162,6 +1162,8 @@
     <PropertyGroup  Condition="'$(AllowPartialPackages)' == 'true' AND '@(_missingPackageFile)' != ''">
       <Description>WARNING: This package is missing files @(_missingPackageFile->'%(TargetPath)/%(Filename)%(Extension)') %0A$(Description)</Description>
     </PropertyGroup>
+
+    <Error Condition="'$(LicenseUrl)' == ''" Text="LicenseUrl property is not defined." />
```

Comment:
The plan so far is to remove the property from the build tools. but I'll go ahead and remove this Error now and then we can add it later when we remove the property